recipes/default.rb

```diff
@@ -235,6 +244,19 @@
   action :create
 end
 
+bind_ip = "0.0.0.0"
+if conda_helpers.bind_services_private_ip
+  bind_ip = my_ip
+end
+
+if node['hops']['nn']['private_ips'].include?(my_ip)
+  # If I'm a NameNode set it to my fqdn
+  nn_address = node['fqdn']
```

Comment:
I don't think it's going to be massively used in the future, but since we have it, could you please handle the case where `install/localhost` is true?

Reply:
Right. Fixed it